samples/add_dvs.py

```diff
@@ -0,0 +1,122 @@
+# Written by Mahesh kumar and Pavan Bidkar
+# GitHub: https://github.com/vmware/pyvmomi
+# Email: kumahesh@vmware.com, pbidkar@vmware.com
+#
+# This code has been released under the terms of the Apache-2.0 license
+# http://opensource.org/licenses/Apache-2.0
+#
+# Description: This sample exports, deletes and imports the
```

Comment:
Provide details of prerequisite required to run the Samples
Looks like it needs virtual switch